packages/compass-components/src/components/spin-loader.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { render, cleanup } from '@testing-library/react';
+
+import SpinLoader from './spin-loader';
+
+function renderLoader() {
+  return render(<SpinLoader size="12px" />);
+}
+
+describe('SpinLoader Component', function () {
+  afterEach(function () {
+    cleanup();
+  });
+
+  it('should show the spinner', function () {
+    renderLoader();
```

Comment:
I have no idea how we'd test this. What's there to check for? It has no text, no sub-elements, no interactivity.. :)